recipes/ifermi/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "ifermi" %}
+{% set version = "0.2.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ifermi-{{ version }}.tar.gz
+  sha256: e350e4dafd95d5bb019af683d7b016cbb2535a62a43b46050d004e924d3664e8
+
+build:
+  number: 0
+  entry_points:
+    - ifermi = ifermi.cli:cli
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
```

Comment:
Upstream says "OS independent". I can't see the build log, why did this fail on Windows?

Reply:
`boltztrap2` is only available on Linux and Mac Os X - still you are right no arch is most likely fine.